src/docs/product/sentry-basics/tracing/trace-view.mdx

```diff
@@ -33,7 +33,8 @@ The transaction view provides a split view similar to the [span view](/product/s
 ### Orphan Traces and Broken Subtraces
 
 In a fully instrumented system, a span in one service will connect to a transaction in a subsequent service. For a variety of reasons, a transaction in a service may not arrive in Sentry. When Sentry encounters these types of transactions within a trace, we link the transactions with a dashed line since they can no longer be directly connected to the root.
```

Comment:
```suggestion
In a fully instrumented system, a span in one service will connect to a transaction in a subsequent service. For a variety of reasons, a transaction in a service may not arrive in Sentry. When Sentry encounters these types of transactions within a trace, the transactions are linked with a dashed line since they can no longer be directly connected to the root, creating an orphan trace.

```

Clarify that this is a case of an orphan trace (I think) and add an extra line to start a new paragraph